src/store/cache/user/hooks.ts

```diff
@@ -355,6 +355,23 @@ export const useTotalDelegates = ({ wallet }: UseTotalDelegatesProps) => {
   return { status, totalDelegates }
 }
 
+/** Returns the total amount delegated inbound to an SP */
+export const useTotalInboundDelegation = ({ wallet }: { wallet: Address }) => {
```

Comment:
useActiveInboundDelegation (total is a scary words that has lost meaning i think)

Reply:
will change

Reply:
or maybe useSummedActiveInboundDelegation? i get that you'd wanna capture the "totaling" of it

Reply:
I think summed is implied by your OG suggestion tbh